appvia/kore
# Fix accounts

GCP Account Management

Adding the account management beta workflow to provision GCP projects on behalf of the teams

### Testing 
- Create a service key from the kore-admin project (service account: kore-admin)
- Provision the secret in kore `kore create secret gcp --from-file=key=/home/jest/Downloads/kore-admin-8b73d30b0ab3.json --force -t kore-admin -d "Appvia Org"`
- Update the examples/gcp.yml with the params and `kore apply -f <file> t kore-admin`
- Create a cluster `kore create cluster <name> -a gke-accounts -p gke-development --show-time`

You should see the project and cluster created
